javascript/node/selenium-webdriver/safari.js

```diff
@@ -110,6 +112,30 @@ class Options extends Capabilities {
     this.options_[TECHNOLOGY_PREVIEW_OPTIONS_KEY] = !!useTechnologyPreview;
     return this;
   }
+
+  /**
+   * Instruct the SafariDriver to use the Safari Automatic Inspection if true.
+   * Otherwise, use the release version of Safari. Defaults to using the release version of Safari.
```

Comment:
Comment doesn't make sense.  Does this feature have site we can point users at  for details?

Reply:
I was commenting that the "Otherwise, use the release version..." line doesn't make sense for this documentation and should be removed.